src/tokenOperations/tokenOperationsFactoryConfig.ts

```diff
@@ -0,0 +1,27 @@
+import BigNumber from "bignumber.js";
+import { Address } from "../address";
+import { IAddress, IChainID, IGasLimit, IGasPrice } from "../interface";
+
+export class TokenOperationsFactoryConfig {
+    chainID: IChainID;
+    minGasPrice: IGasPrice = 1000000000;
+    minGasLimit = 50000;
+    gasLimitPerByte = 1500;
+    gasLimitIssue: IGasLimit = 60000000;
+    gasLimitESDTLocalMint: IGasLimit = 300000;
+    gasLimitESDTLocalBurn: IGasLimit = 300000;
+    gasLimitSetSpecialRole: IGasLimit = 60000000;
+    gasLimitPausing: IGasLimit = 60000000;
+    gasLimitFreezing: IGasLimit = 60000000;
+    gasLimitESDTNFTCreate: IGasLimit = 3000000;
+    gasLimitESDTNFTUpdateAttributes: IGasLimit = 1000000;
+    gasLimitESDTNFTAddQuantity: IGasLimit = 1000000;
+    gasLimitESDTNFTBurn: IGasLimit = 1000000;
+    gasLimitStorePerByte: IGasLimit = 50000;
+    issueCost: BigNumber.Value = "50000000000000000";
```

Comment:
not valid for internal testnets, where is starts with 5 xegld by default.

Reply:
Yes, in that case, the developer has to create a new config object, alter the field, and only then pass it to the factory.